### DISCUSSION TITLE
[OP Stack Deployment] Private op-stack blockchain which syncs to mainnet.

### DISCUSSION THREAD

**Discussion by baahkusi** [ethereum-optimism/developers#28] [Deploying the OP Stack] [answered]
### Issue Description

I'm working on a project that requires the use of a Private Blockchain. I want to use opstakck and deploy as a private optimism instance. 

Basically what I want to achieve is a private block chain which syncs to mainnet. This will help me keep the data private whilst benefiting from the security of both mainnet and the private op stack instance. 

Is that possible? 

The existing option is to deploy a private blockchain like besu or fabric. But it doesn't benefit from syncing to mainnet.

**Comment by opfocus** [comment]
I think it's possible to control access permissions for nodes. However, decoding data on L1 is not something typically done. Nevertheless, it can still be decoded. I'm not very familiar with this issue. cc @sbvegan

**Comment by sbvegan** [comment, marked as answer]
This is not currently supported. With these requirements, you would need to post batch data somewhere other than mainnet.

**Comment by sbvegan** [comment]
Hey @baahkusi, just following up here. Can you please mark the response that answered your question?
